### DISCUSSION TITLE
Breaking Changes 5.x - List

### DISCUSSION THREAD

**Discussion by Luke-Sikina** [cBioPortal/cbioportal#9511] [General]
Here are some breaking changes we need to make in 5.x

- Remove `functional_impact_score` from `mutation_event` in the DB. Also remove field from corresponding API objects
- Upgrade to Java 17
- Add `sample_id` field to `clinical_event`

**Comment by inodb** [comment]
@Luke-Sikina I think we're gonna put the structural variant work as 5.x since it involves a lot of changes. So maybe this is more for 6.x? Or would you prefer to try and roll these changes out as part of that?

**Reply by Luke-Sikina** [reply]
What's the timeline for 5.x? I'd really like to sneak java 17 in there if I can, but it's probably 2 weeks of work. `functional_impact_score` would be another good one to add since it should be low impact
